cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml

```diff
@@ -117,6 +117,14 @@
       <artifactId>twitter4j-stream</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-mapreduce-client-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
```

Comment:
we can't package it, as the user may want to use a specific version of the mysql connector, or may not want mysql at all.  The built-in source and sink doesn't actually require this since it uses jdbc, though it requires some sort of connector jar.  This is one of the reasons the plugin work is required.